### PR TITLE
Footermod mm

### PR DESCRIPTION
**Pull request by snschau**
# Summary of changes
Modify Custom Footer to add the social media icons for ovc footers 
check to show that non ovc footers are not effected

## Frontend
- Add React Icons to packages
- Add footer-social.js (to blocks/ovc)
  - custom code to add social media icons
- Modified customFooter.js to check if unit tag is OVC and if so - add social icons below the image.
- Add social-icons.css - to remove some styles that were interfering with how icons looked  

 
## Backend
- None

# Test Plan

https://ovccontenthub.gatsbyjs.io/ovc/gceb3 icons at the bottom should work as on https://ovc.uoguelph.ca/
https://ovccontenthub.gatsbyjs.io/test-media-text-in-section - a test page to show that if not an ovc footer - icons don't display
https://ovccontenthub.gatsbyjs.io/cancer-to-delete (or any page that does not have a footer) no change or footer elements added

Note: build below same pages - but using api.live data (update Mar 1st 9:40 am):
https://build-f9df32a8-a745-4c60-abb6-d7f8d83bc7e9.gtsb.io/

https://build-f9df32a8-a745-4c60-abb6-d7f8d83bc7e9.gtsb.io/gceb3
https://build-f9df32a8-a745-4c60-abb6-d7f8d83bc7e9.gtsb.io/test-media-text-in-section

regular pages with no tags for a custom footer - https://build-f9df32a8-a745-4c60-abb6-d7f8d83bc7e9.gtsb.io/choose-u-of-g

### CODE REVIEW

File: src/components/blocks/ovc/footerSocial.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    <a href="http://www.facebook.com/ontvetcollege" className="text-dark-social">
+      <span className="sr-only">Connect with OVC on Facebook</span>
+      <FaFacebookSquare style={{fontSize: "4.8rem"}} />
+    </a>
+    <a href="http://twitter.com/OntVetCollege/" className="text-dark-social">
+      <span className="sr-only">Connect with OVC on Twitter</span>
+      <FaTwitterSquare style={{fontSize: "4.8rem"}} />
+    </a>
+    <a href="http://www.youtube.com/user/OntarioVetCollege" className="text-dark-social">
+      <span className="sr-only">Connect with OVC on YouTube</span>
```
Comment (mmafe) [inline src/components/blocks/ovc/footerSocial.js:33]: I think Bootstrap 5 has updated their classes for sr-only to be visually-hidden. Not a deal breaker though. https://getbootstrap.com/docs/5.1/helpers/visually-hidden/

File: src/components/blocks/ovc/footerSocial.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    <h4 className="mt-4 text-dark">Stay Connected</h4>
+    <a href="https://instagram.com/ontvetcollege/" className="text-dark-social">
+      <span className="sr-only">Connect with OVC on Instagram</span>
+      <FaInstagram style={{fontSize: "4.8rem"}}/>
+    </a>
+    <a href="https://www.linkedin.com/school/ontario-veterinary-college/" className="text-dark-social">
+      <span className="sr-only">Connect with OVC on LinkedIn</span>
+      <FaLinkedin style={{fontSize: "4.8rem"}}/>
+    </a>
+    <a href="http://www.facebook.com/ontvetcollege" className="text-dark-social">
```
Comment (mmafe) [inline src/components/blocks/ovc/footerSocial.js:24]: Can we use https instead of http?

File: src/components/blocks/ovc/footerSocial.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      <span className="sr-only">Connect with OVC on Facebook</span>
+      <FaFacebookSquare style={{fontSize: "4.8rem"}} />
+    </a>
+    <a href="http://twitter.com/OntVetCollege/" className="text-dark-social">
```
Comment (mmafe) [inline src/components/blocks/ovc/footerSocial.js:28]: Can we use https instead of http?

File: src/components/blocks/ovc/footerSocial.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      <span className="sr-only">Connect with OVC on Twitter</span>
+      <FaTwitterSquare style={{fontSize: "4.8rem"}} />
+    </a>
+    <a href="http://www.youtube.com/user/OntarioVetCollege" className="text-dark-social">
```
Comment (mmafe) [inline src/components/blocks/ovc/footerSocial.js:32]: Can we use https instead of http?

**Review by mmafe — APPROVED** [review]
Added a few minor suggestions (https instead of http and possibly using the Bootstrap 5 class for visually hiding things)

File: src/components/blocks/ovc/footerOVC.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types';
```
Comment (mmafe) [inline src/components/blocks/ovc/footerOVC.js:1]: Scott, forgive me! I just looked at my own code and 've been naming all my site-specific components with dashes. 🤣 You can totally switch it back to footer-ovc.js if you want.

**Review by tqureshi-uog — APPROVED** [review]
No issues found - approved.

**Pull request merged**